### ISSUE TITLE
[Documentation]: Need enough documentation

### ISSUE THREAD

**Issue by gekkowrld** [codetrybe/community#10]
### What would you like to share?

This repo needs a documentation, i.e editing the relevant .md files.

**Comment by tpauldike** [comment]
Do you want to suggest certain contents?

**Comment by gekkowrld** [comment]
Yeah, Include how the community values, how to contribute,Introduction for everyone (or yourself for the mean time) and other things that seem useful to the community. It is really about your freedom to think and write something good.

**Comment by tpauldike** [comment]
Soon
